core/src/test/java/nl/nn/adapterframework/http/HttpSenderTest.java

```diff
@@ -1034,10 +1073,7 @@ public void unsupportedScheme() throws Throwable {
 		sender.configure();
 		sender.open();
 
-		sender.sendMessageOrThrow(input, pls).asString();
-
-		// We expect sendMessage to throw expection
-		assertTrue(false);
+		assertThrows(SenderException.class, () -> sender.sendMessageOrThrow(input, pls));
```

Comment:
Het zou handiger zijn als er al een ConfigurationException komt bij `sender.configure()`

Reply:
Het is een param, dus dat kan niet want die resolved hij pas runtime